src/Test.UnitTests.Sarif.Multitool.Library/TestData/MergeCommand/Inputs/FileNameOnly.sarif

```diff
@@ -0,0 +1,360 @@
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "docker.io/fluxcd/flux:1.24.1 (alpine 3.14.2): openssh-client-common-8.6_p1-r2 CVE-2021-41617",
+          "ruleIndex": 3,
+          "level": "error",
+          "message": {
+            "text": "sshd in OpenSSH 6.2 through 8.x before 8.8, when certain non-default configurations are used, allows privilege escalation because supplemental groups are not initialized as expected. Helper programs for AuthorizedKeysCommand and AuthorizedPrincipalsCommand may run with privileges associated with group memberships of the sshd process, if the configuration specifies running the command as a different user."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "docker.io/fluxcd/flux:1.24.1 (alpine 3.14.2): openssh-client-default-8.6_p1-r2 CVE-2021-41617",
+          "ruleIndex": 4,
+          "level": "error",
+          "message": {
+            "text": "sshd in OpenSSH 6.2 through 8.x before 8.8, when certain non-default configurations are used, allows privilege escalation because supplemental groups are not initialized as expected. Helper programs for AuthorizedKeysCommand and AuthorizedPrincipalsCommand may run with privileges associated with group memberships of the sshd process, if the configuration specifies running the command as a different user."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "docker.io/fluxcd/flux:1.24.1 (alpine 3.14.2): openssh-keygen-8.6_p1-r2 CVE-2021-41617",
+          "ruleIndex": 5,
+          "level": "error",
+          "message": {
+            "text": "sshd in OpenSSH 6.2 through 8.x before 8.8, when certain non-default configurations are used, allows privilege escalation because supplemental groups are not initialized as expected. Helper programs for AuthorizedKeysCommand and AuthorizedPrincipalsCommand may run with privileges associated with group memberships of the sshd process, if the configuration specifies running the command as a different user."
+          },
+          "locations": [{
+            "physicalLocation": {
+              "artifactLocation": {
+                "uri": "docker.io/fluxcd/flux:1.24.1",
+                "uriBaseId": "ROOTPATH"
+              }
+            }
+          }]
+        },
+        {
+          "ruleId": "usr/local/bin/kustomize: github.com/ulikunitz/xz-v0.5.5 CVE-2021-29482",
```

Comment:
>usr/local/bin/kustomize: github.com/ulikunitz/xz-v0.5.5 CVE-2021-29482" [](http://example.com/codeflow?start=21&length=71)

Is this really the rule id for this tool?  this is extremely atypical. was this discussed during the SARIF export dev process?

Reply:
Is this really the rule id for this tool ---- yes, this is from the user who reported the merge bug, it is not generated  from our tool.